File: com/jmonclus/problems/Problem0009.java
package com.jmonclus.problems;

/**
 * Problem url: https://projecteuler.net/problem=9
 * 
 * @author Josep Monclus
 */
public class Problem0009 {
    public static void main(String[] args) {
        int expectadValue = 1000;

        for(int i = expectadValue; i > 0; i--) {
            for(int j = expectadValue - i; j > 0; j--) {
                for(int l = expectadValue - i - j; l > 0; l--) {
                    if(i * i == (j * j) + (l * l)){
                        if(i + j + l == expectadValue) {
                            System.out.println(i + " - " + j + " - " + l);
                            System.out.println(i * j * l);
                        }
                    }
                }
            }
        }
    }
}

File: python/problem_0009.py

# Problem url: https://projecteuler.net/problem=9

def is_pitagorean(a, b, c):
    if a < b and b < c and a**2 + b**2 == c**2:
        return True
    else:
        return False

expected_sum = 1000

for a in range(1, expected_sum + 1):
    for b in range(a + 1, expected_sum + 1 - a):
        c = expected_sum - a - b
        if is_pitagorean(a, b, c):
            print('Pitagorean found: ', a, ' - ', b, ' - ', c)
            print('Product of this is: ', a * b * c)